crates/driver/src/tests/setup/driver.rs

```diff
@@ -76,12 +79,7 @@ pub fn solve_req(test: &Test) -> serde_json::Value {
             "protocolFees": match quote.order.kind {
                 order::Kind::Market => json!([]),
                 order::Kind::Liquidity => json!([]),
-                order::Kind::Limit { .. } => json!([{
-                    "surplus": {
-                        "factor": 0.0,
-                        "maxVolumeFactor": 0.06
-                    }
-                }]),
+                order::Kind::Limit { .. } => json!([quote.order.fee_policy.to_json_value()]),
```

Comment:
Don't we expect this in all variants? How will this work once we have fees for market orders?

Reply:
Enabled for all variants.

Reply:
upd: reverted the change for now since it affects other tests.

Reply:
So how will this work once we have fees for market orders as well?

Reply:
Market order are planned to be deprecated by the 15th of March, as planned by BD. We also won't test the fee for in-market limit orders by then.